tools/install/composer.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What is `tools/install`?

Reply:
That was by accident. Removed in eeae688f32